.github/workflows/github_actions_aws_win_py32.yml

```diff
@@ -0,0 +1,20 @@
+name: nidaqmxbot_win_32
+
+on:
+  pull_request:
+    branches:
+      - master
+jobs:
+  run-system-tests-python-32bit:
+    runs-on:
+      - self-hosted
+      - windows
+      - x64
+      - rdss-nidaqmxbot-win-10-py32
```

Comment:
This line and the name are the only differences between these files, so we should combine them and use a matrix.

https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs

Something like this:
```
runs-on:
  - self-hosted
  - rdss-nidaqmxbot-${{ matrix.configuration }}
strategy:
  matrix:
    configuration: [win-10-py32, win-10-py64]
```

I don't think it's necessary to specify runs-on: windows and runs-on: x64. Restricting to a specific runner should constrain the OS and CPU.

Reply:
Thanks for the information! Updated.